docs/reference/manifest/control.md

```diff
@@ -31,9 +31,12 @@ A button performs a single action when the user selects it. It can either execut
 |  [Supertip](supertip.md)  | Yes |  The supertip for the button.    |
 |  [Icon](icon.md)      | Yes |  An image for the button.         |
 |  [Action](action.md)    | Yes |  Specifies the action to perform.  |
+|  [Enabled](enabled.md)    | No |  Specifies whether the control is enabled when the add-in launches.  |
```

Comment:
Are these in any particular order? If they can be alphabetized, please consider doing that for this table.

Reply:
@AlexJerabek This is the required order.